### DISCUSSION TITLE
Not able to use $.getJson on Breeze Enabled pages

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#178] [1. Help]
Hi,
I am unable to use $.getJson on pages which have breeze enabled. Can you provide any solution to use it in Breeze?
Error: Uncaught TypeError: $.getJson is not a function

Thanks

**Comment by vovayatsyuk** [comment]
`$.getJson` is just a wrapper around `$.ajax`. Use [`$.ajax`](https://breezefront.com/docs/ajax) instead.
